.github/workflows/build-checks.yml

```diff
@@ -60,6 +60,8 @@ jobs:
       - name: Build and test with Maven
         env:
           AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
-        run: mvn --batch-mode --update-snapshots clean install -P e2e
+        run: |
+              mvn -f tests/end-to-end/pom.xml versions:update-properties
```

Comment:
Shouldn't that be executed only during scheduled pipeline?
Latest version should not be checked during the pull request pipeline.